app/assets/javascripts/angular/helpers/sc2.js

```diff
@@ -258,9 +258,13 @@ for (var expansion_tag in Sc2.armyUnits) {
 
 // I'd rather stuff these here, than in Match
 
-Sc2.frameToTime = function(frame) {
-  minute = Math.floor(frame / (60 * 16));
-  second = Math.floor((frame / 16) % 60).toString();
+Sc2.frameToTime = function(frame, expansion) {
+  fps = 16;
+  if (expansion >= 2) {
+    fps *= 1.4;
+  }
+  minute = Math.floor(frame / (60 * fps));
+  second = Math.floor((frame / fps) % 60).toString();
```

Comment:
just to doublecheck carefully:
Pre-LotV: 960 frames = 1:00 on the game clock = 0:43 on a real clock
LotV: 960 frames = 0:43 on the game clock = 0:43 on a real clock

so, yeah there used to be 16 frames per game second, and now there's ~16 * 1.4 frames per game second.  looks good to me.